[2022-02-08 22:10:15,145] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.spark_job 2022-02-08T22:10:14.344486+00:00 [queued]>
[2022-02-08 22:10:15,181] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.spark_job 2022-02-08T22:10:14.344486+00:00 [queued]>
[2022-02-08 22:10:15,182] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 22:10:15,182] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-02-08 22:10:15,182] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 22:10:15,224] {taskinstance.py:1107} INFO - Executing <Task(SparkSubmitOperator): spark_job> on 2022-02-08T22:10:14.344486+00:00
[2022-02-08 22:10:15,237] {standard_task_runner.py:52} INFO - Started process 54 to run task
[2022-02-08 22:10:15,248] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'hello_world', 'spark_job', '2022-02-08T22:10:14.344486+00:00', '--job-id', '20', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmpyk4don3m', '--error-file', '/tmp/tmp96l9xrox']
[2022-02-08 22:10:15,249] {standard_task_runner.py:77} INFO - Job 20: Subtask spark_job
[2022-02-08 22:10:15,388] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.spark_job 2022-02-08T22:10:14.344486+00:00 [running]> on host e4b67ec693bf
[2022-02-08 22:10:15,555] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=spark_job
AIRFLOW_CTX_EXECUTION_DATE=2022-02-08T22:10:14.344486+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-08T22:10:14.344486+00:00
[2022-02-08 22:10:15,616] {base.py:69} INFO - Using connection to: id: spark_default. Host: spark://spark, Port: 7077, Schema: , Login: ***, Password: ***, extra: {}
[2022-02-08 22:10:15,618] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name test --verbose /opt/spark/app/test.py
[2022-02-08 22:10:15,849] {spark_submit.py:514} INFO - JAVA_HOME is not set
[2022-02-08 22:10:15,869] {taskinstance.py:1501} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1157, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1331, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1361, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 182, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 445, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name test --verbose /opt/spark/app/test.py. Error code is: 1.
[2022-02-08 22:10:15,874] {taskinstance.py:1544} INFO - Marking task as FAILED. dag_id=hello_world, task_id=spark_job, execution_date=20220208T221014, start_date=20220208T221015, end_date=20220208T221015
[2022-02-08 22:10:15,952] {local_task_job.py:151} INFO - Task exited with return code 1
